include/aws/cal/hash.h

```diff
@@ -84,6 +90,20 @@ AWS_CAL_API int aws_sha256_compute(
     struct aws_byte_buf *output,
     size_t truncate_to);
 
+/**
+ * Computes the sha1 hash over input and writes the digest output to 'output'.
+ * Use this if you don't need to stream the data you're hashing and you can load
+ * the entire input to hash into memory. If you specify truncate_to to something
+ * other than 0, the output will be truncated to that  number of bytes. For
+ * example if you want a SHA1 digest as the first 16 bytes, set truncate_to
+ * to 16. If you want the full digest size, just set this to 0.
+ */
```

Comment:
Trivial: single space between 'that' and 'number', and then comma after "For example".